pydantic/fields.py

```diff
@@ -3,6 +3,7 @@
 from typing import (
     TYPE_CHECKING,
     Any,
+    Callable,
```

Comment:
if the below is `TypingCallable`, what is this? Maybe good to give this an alias.

Reply:
I decided to add a `NoArgCallable` directly in the local `typing` module. Makes things clearer imo